packages/start/plugin.js

```diff
@@ -285,6 +286,30 @@ function solidStartConfig(options) {
   };
 }
 
+function find(locate, cwd) {
+  cwd = cwd || process.cwd();
+  if (cwd.split(path.sep).length < 2) return undefined;
+  const match = fs.readdirSync(cwd).find((f) => f === locate);
+  if (match) return match;
+  return find(locate, path.join(cwd, ".."));
+}
+
+function detectAdapter() {
```

Comment:
Is there a way to ensure that this doesn't mistakenly use a package like `solid-start-form-lib` as the adapter? People might come up with package names like that. We can have a system of reading the package.json in any such package and if it has a specific config that tells us it's an adapter, we use it? What do you think?

Reply:
Sounds like a good idea, to place a "solid.adapter": true in the package, I'll try to update the pull request!